pwnagotchi/locale/de/LC_MESSAGES/voice.po

```diff
@@ -174,10 +174,10 @@ msgstr "Cool, wir haben {num} neue Handshake{plural}!"
 
 #, python-brace-format
 msgid "You have {count} new message{plural}!"
-msgstr "Cool, wir haben {num} neue Handshake{plural}!"
```

Comment:
I think "Handshakes" would be the better word, even for german pwnagotchi users.